scripts/olm-create-bundle.sh

```diff
@@ -127,6 +127,10 @@ pushd $tmp_kustomize_config_dir/controller 1>/dev/null
 kustomize edit set image controller="$AWS_SERVICE_DOCKER_IMG"
 popd 1>/dev/null
 
+# remove crd/common from bases to prevent inclusion of AdoptedResource CRD from being generated in the bundle directory
+sed -i '' '/bases:/d' $tmp_kustomize_config_dir/crd/kustomization.yaml
+sed -i '' '/- common/d' $tmp_kustomize_config_dir/crd/kustomization.yaml
```

Comment:
I may be very wrong here but I have never seen this usage of `sed` command before.

I ran these commands on a dummy file locally and these commands did not cause any changes in my dummy file on zsh shell.

Can you please tell me the environment setting, where I can test this command to validate that it works?

Reply:
The integration tests run on debian image https://github.com/aws-controllers-k8s/test-infra/blob/main/prow/jobs/images/Dockerfile.olm-test

Can you please make sure the sed command you are using works on debian platform to get s3-olm-test to pass. : )